ui/lib/core/addon/components/form-field.js

```diff
@@ -92,11 +92,11 @@ export default class FormFieldComponent extends Component {
 
     // here we replicate the logic in the template, to make sure we don't change the order in which the "ifs" are evaluated
     if (options?.possibleValues?.length > 0) {
-      if (options?.editType === 'checkboxList') {
-        return true;
-      } else {
-        // we still have to migrate the `radio` and `select` use cases
+      // we still have to migrate the `radio` use case
+      if (options?.editType === 'radio') {
```

Comment:
An aside, a good component to check for editType radio in this use case is during MFA. Navigate to `http://localhost:4200/ui/vault/access/mfa/methods/create?type=totp` and under Algorithm you'll see a radio with possibleValue 
![image](https://github.com/user-attachments/assets/566ed255-10ca-4a3a-ba0d-c2bf7a16a033)

Reply:
I've added this comment to the `radio` ticket. Thanks!